### ISSUE TITLE
HSIO Configuration and Graphic outputs

### ISSUE THREAD

**Issue by acruxcz** [LattePandaTeam/LattePanda-Mu#9]
Hi,

we just design our product using your module, so:
- what about PCIe BIOS? We prefer maximum number of PCI and 2 or 4 USB 3. What is a planed configuration for that bios?
- We strongly prefer using only DP/eDP ports. There is any possibility to use all 	TCP ports for DP? According ADL-N datasheet, it should be possible.... 

Thanks.

**Comment by alphaarea** [comment]
- You can configure the PCIe and USB3 ports to the extent that the N100 CPU hardware allows, as shown below.
- The video outputs can all be configured as DP connectors, the circuitry for DP is simpler than for HDMI.

![image](https://github.com/LattePandaTeam/LattePanda-Mu/assets/26252565/b0dc78bd-6ea0-4880-ba48-664af3cf4994)

**Comment by acruxcz** [comment]
But this require specific bios for that configuration, right?

**Comment by alphaarea** [comment]
Yes, to change the port configuration you must change the BIOS binary, which cannot be done through the BIOS setup menu.

You can request a test BIOS by sending specific configuration requirements to techsupport@dfrobot.com.

**Comment by DiodeDrop** [comment]
> 
>     * The video outputs can all be configured as DP connectors, the circuitry for DP is simpler than for HDMI.
>

Do you know which lanes are which on TCP0 when configured for DP?
e.g.
TCP0_TX0 --> LANE0
TCP0_TXRX0 --> LANE1
etc.

Or, would you have a reference schematic for DP on the Mu?

**Comment by DiodeDrop** [comment]
> 
>     * The video outputs can all be configured as DP connectors, the circuitry for DP is simpler than for HDMI.
>

Do you know which lanes are which on TCP0 when configured for DP?
e.g.
TCP0_TX0 --> LANE0
TCP0_TXRX0 --> LANE1
etc.

Or, would you have a reference schematic for DP on the Mu?
